pages/roadmap.js

```diff
@@ -0,0 +1,76 @@
+              </p>
+              <p className="mt-3 text-lg leading-7 text-gray-500 lg:pr-4">
+                It all starts with a vision. This is where we want to see AsyncAPI by 2026.
+              </p>
+            </div>
+
+            <div className="relative mt-20">
+              <div className="grid lg:grid-cols-3 lg:gap-8">
+                <div>
+                  <h2 className="font-semibold mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    AsyncAPI unifies all the API specifications
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Nobody does only event-driven architectures. Most people complement them with REST (OpenAPI), GraphQL, and/or RPC APIs. <strong>We want people to use the AsyncAPI specification and tooling together with their existing OpenAPI, GraphQL, and gRPC definitions.</strong> For that purpose, our specification and tools will need to understand and leverage many other specifications and tools.
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    This is not about reinventing the wheel or creating yet another spec to solve the same problems but to integrate with the existing tools instead.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    API development experience is seamless
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    We want to make the development experience super seamless, from idea to production. <strong>Users should be able to create their first API within minutes without prior knowledge of AsyncAPI, the communication protocol, or anything else that's not business logic.</strong>
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Furthermore, making sure production behavior and documentation are always aligned, independently of the programming language or framework of choice.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    The AsyncAPI community grows 400%
```

Comment:
Why is the community expected to grow by 400% and not some other amount? How is this calculated and why so?

Reply:
This is not expected but instead, it's a goal. The number is just a reference to what I think would be enough for us to handle such a workload.

Reply:
Thanks, I got it :)